pkg/bbgo/order_executor_general.go

```diff
@@ -144,19 +144,21 @@ func (e *GeneralOrderExecutor) BindProfitStats(profitStats *types.ProfitStats) {
 	})
 }
 
-func (e *GeneralOrderExecutor) Bind() {
```

Comment:
maybe use a setter for this: DisableNotify()

Reply:
Using DisableNotify will still add additional callbacks on tradeCollector, unless you force users to call this before Bind().

Reply:
yes, we want to by default enable the notification. It's less chance that one would like to disable the notification

Reply:
done.